src/utils/renameImport.js

```diff
@@ -36,7 +36,9 @@ export function updateImport (j, root, { oldValue, newValue }) {
   root
     .find(j.ImportDeclaration)
     .filter(p => {
-      return p.parent.node.type === 'Program' && p.get('specifiers', 0, 'type').value !== 'ImportNamespaceSpecifier';
+      return  p.parent.node.type === 'Program' &&
+              p.get('specifiers', 0, 'type').value !== 'ImportNamespaceSpecifier' &&
+              p.value.source.value === 'can' || p.value.source.value === '//unpkg.com/can@5/everything.mjs';
```

Comment:
Regex match like `/\/\/unpkg.com\/can@5(\.[0-9]+)?(\.[0-9]+)?\/eveything.mjs/` would be better here.